netlify/landings/prism.yaml

```
@@ -0,0 +1,79 @@
+path: /prism
+title: Open Source Mock Server
+color: purple-darker
+relatedTags:
+  - blog-mocking
+hero:
+  subtitle: Build APIs and accelerate API development with instant mock servers
+  title: Write the spec. Prism will do the rest.
+featureSection:
+  actionBar:
+    buttons:
+      - href: 'https://github.com/stoplightio/prism'
+        title: Try Prism now
```

```suggestion
        title: Get Started
```
"Try" makes it sound like a trial.